lib/active_model/serializer.rb

```diff
@@ -112,10 +112,25 @@ def self.digest_caller_file(caller_line)
       Digest::MD5.hexdigest(serializer_file_contents)
     end
 
+    # TODO(beauby): Cache this.
+    def self.serializer_lookup_chain_for(klass)
```

Comment:
Looks pretty much self-explanatory to me. What kind of comment would you add that would not be direct paraphrasing of the code?